ts/components/dialog/OnionStatusPathDialog.tsx

```diff
@@ -65,6 +66,15 @@ const OnionPathModalInner = () => {
     },
   ];
 
+  // Ensure we can always find the GeoLite2 database, regardless of whether
+  // this is a dev or a prod build.
+  const binPath = (process.env.NODE_APP_INSTANCE || '').startsWith('devprod')
```

Comment:
this is being done in the render thread, it should not be, it should be done in a worker so we dont need to pull on the fs package via npm, and we can use electron's built in calls for file access